### PR TITLE
PREAPPS-2077: Forward event from Calendar context menu

### PR DESCRIPTION
**Pull request by craigdanj**

### CODE REVIEW

File: src/normalize/entities.ts [hunk 1]
```diff
@@ -349,6 +350,11 @@ const ForwardMessageInput = new Entity({
 	su: 'subject'
 });
 
+export const ForwardAppointmentInfo = new Entity({
+	m: ['message', ForwardMessageInput],
+	exceptId: ['exceptId', InstanceDate]
```
Comment (SantoshKShetty) [inline src/normalize/entities.ts]: `exceptId` is not required to be included here.

Comment (craigdanj) [reply src/normalize/entities.ts]: not sure I understood. We have to pass exceptId when we want to forward an instance right?

Comment (craigdanj) [reply src/normalize/entities.ts]: As discussed with you.. this is needed hence we need to keep it.

Comment (SantoshKShetty) [reply src/normalize/entities.ts]: Sorry. Missed above `InstanceDate`. Thanks @craigdanj

**Pull request merged**